client/app/services/notification.js

```diff
@@ -0,0 +1,22 @@
+import notification from 'antd/lib/notification';
+
+notification.config({
+  placement: 'bottomRight',
+  duration: 3,
+});
+
+const updatedActions = [
+  'success',
+  'error',
+  'info',
+  'warning',
+  'warn',
+].reduce((acc, action) => ({
+  ...acc,
+  [action]: (message, description = null, props = null) => notification[action]({ ...props, message, description }),
+}), null);
+
+export default { // export Ant's notification and replace actions
+  ...notification,
+  ...updatedActions,
+};
```

Comment:
For notification consistency, consider some constants to be used across components:
```js
{
  SAVE_SUCCESS: 'Saved',
  SAVE_ERROR: 'Failed saving',
  ...
}
```
```js
notification.error(notification.SAVE_ERROR, 'Sth went wrong');
```

Reply:
This is an interesting idea, but not sure if we really need it. At least, we should check if we have two or more messages with the same titles.

Reply:
We do. Many.
And some others are variations which I think is worth consolidating.

Reply:
But, it's definitely out-of-scope here. Future thinking.

Reply:
We need this for sure :+1:. I think for now we can add both `SAVE_SUCCESS` and `SAVE_ERROR` as they are a direct change in code and wouldn't be out-of-scope. This will at least define a future pattern and we can later modify existing notifications, WDYT?

Reply:
I think it has value for ux, but I'd rather land this as is so we could move on to the next angular2react component.